sql/parser/builtins.go

```diff
@@ -451,6 +466,7 @@ var Builtins = map[string][]Builtin{
 		Builtin{
 			Types:      ArgTypes{},
 			ReturnType: TypeBytes,
+			category:   "ID Generation",
```

Comment:
don't you want a type for this with a bunch of typed constants? this looks prone to typos.